.github/files/phpcompatibility-dev-phpcs.xml

```diff
@@ -14,5 +14,8 @@
 		<exclude name="PHPCompatibility.FunctionUse.NewFunctions.sodium_crypto_box_sealFound" />
 		<exclude name="PHPCompatibility.Classes.NewClasses.sodiumexceptionFound" />
 		<exclude name="PHPCompatibility.FunctionUse.NewFunctions.is_countableFound" />
+
+		<!-- Analyzer doesn't follow the same requirements. -->
+		<exclude-pattern>/projects/packages/analyzer/*</exclude-pattern>
```

Comment:
This should probably be changed for the changes in #22117.

Probably put it in `projects/packages/analyzer/.phpcs.dir.phpcompatibility.xml`